dato/js/recview.js

```diff
@@ -0,0 +1,158 @@
+        h2.innerText = 'Recommended Resources';
+        const root = document.createElement('div');
+        root.id = 'root';
+        const list = document.createElement('ul');
+        list.id = 'list';
+
+        root.appendChild(list);
+        section.appendChild(h2);
+        section.appendChild(root);
+        $('#section-').after(section);
+
+        function feedback(event){
+          event.preventDefault();
+          var uuid = resp.data.uuid;
+          var itemId = event.target.href;
+          onClick(uuid, itemId);
+        }
+        for (let i in resp.data.response) {
+          let listEl = document.createElement('li');
+          let link = document.createElement('a');
+          link.className = 'recommendation';
+          link.href = resp.data.response[i].page_url;
+          link.innerText = resp.data.response[i].title;
+          link.addEventListener('click', feedback);
+          listEl.appendChild(link);
+
+          document.getElementById('list').appendChild(listEl);
+        }
+
+        // cb event for mouse over event
+        // tracks when user first hovers over recommender section
+        // sends feedback with data
+        function mouseOverCB(event) {
+          console.log(resp.data.uuid);
+
+          // once we know the schema for what to send back on mouseOver the recommendations
+          // uncomment code and set data
+          var data = {
+            'viewed': 1
+          };
+          client.feedback(resp.data.uuid, { data }, function(err, resp) {
+            if(err) {
+              console.log(err);
+            }
+            console.log(resp);
+          });
+        }
+        $( 'ul#list' ).one( "mouseover", mouseOverCB );
+      }
+
+      // onClick event for links in recommender list
+      // parameters: uuid, and itemId
+      // calls feedback funciton in PS client
+      function onClick(uuid, itemId) {
+        console.log("onClick function");
+        console.log(uuid);
+        console.log(itemId);
+        // Calls client feedback function and sends uuid and feedbackData
+        var data = {'itemId': itemId, 'click': 1};
+        client.feedback(uuid, { data }, function(err, resp) {
+          if(err) {
+            console.log(err);
+          }
+          console.log(resp);
+          window.location.replace(itemId);
+        });
+      }
+    }
+
+    // gets snowplowId from cookie and constructs url to send in query
+    // Once we have schema for data sent in query to endpoint we need to add snowplowId
+    let snowplowId = getSnowplowDuid();
+
+    // TODO: If no snowplotId is found, send -1 as a user id. It would be better
+    // if getSnowplowDuid reliably returned a valid user id.
+    if (!snowplowId) {
+      snowplowId = -1;
+    }
+
+
+    // Construct full URL using the window's pathname.
+    let url = window.location.pathname;
+    url = url.replace('/learn/userguide/', 'https://dato.com/learn/userguide/');
+    if (url.includes('_book')) {
```

Comment:
Good point, that makes sense. Please update the comment above to include the motivation vs. just the mechanism -- something like "normalize the URL to a deployed dato.com userguide URL so we get better recommendations" rather than "construct full URL".